src/status_im/node/core.cljs

```diff
@@ -73,7 +73,8 @@
    :Nameserver     "1.1.1.1"
    :PeerExchange   true
    :Port           0
-   :UDPPort        0})
+   :UDPPort        0
+   :Nameserver     "1.1.1.1"})
```

Comment:
Hi key :Nameserver already exists?

Reply:
you probably rebased PR, cammellos just added it hour ago
https://github.com/status-im/status-mobile/commit/14c9a7c6ac003542099e0436b59021b1ba9d60ad

Reply:
I see, thanks